src/platforms/common/performance/instrumentation/performance-metrics.mdx

```diff
@@ -36,23 +36,24 @@ Sentry's SDKs support sending performance metrics data to Sentry. These are nume
 In addition to automatic performance metrics, the SDK supports setting custom performance metrics on transactions. This allows you to define metrics that are important to your application and send them to Sentry.
 
 To set a performance metric, you need to supply the following:
+
 - name (`string`)
 - value (any numeric type - `float`, `integer`, etc.)
-- unit (`string`, can be empty or the string `'none'`). 
+- unit (`string`, can be empty or the string `'none'`).
 
 Sentry supports adding arbitrary custom units, but we recommend using one of the [supported units listed below](./#supported-performance-metric-units).
 
 <PlatformContent includePath="performance/custom-performance-metrics" />
 
-<Alert level="warning" title="Note">
+<Note>
```

Comment:
@lobsterkatie - Mostly just need confirmation that it's okay for me to change this to a regular note. It doesn't seem to warrant red background.

Reply:
Oh, yeah, whoops. I was figuring "warning" was probably going to be yellow. Red does seem excessive.